src/App.tsx

```diff
@@ -47,7 +47,16 @@ const App: FC = () => {
 
   useEffect(() => {
     if (CLOUD && isFlagEnabled('rudderstackReporting')) {
-      load(RUDDERSTACK_WRITE_KEY, RUDDERSTACK_DATA_PLANE_URL)
+      try {
+        load(RUDDERSTACK_WRITE_KEY, RUDDERSTACK_DATA_PLANE_URL)
+      } catch {
+        console.error(
+          'Error loading Rudderstack with wk: ',
+          RUDDERSTACK_WRITE_KEY,
+          ' at: ',
+          RUDDERSTACK_DATA_PLANE_URL
+        )
```

Comment:
I think we should report this error to Honeybadger so we can see it happening on our user's browsers

Reply:
check out the `reportErrorThroughHoneyBadger` function

Reply:
Good call, added